src/Mod/Sketcher/Gui/TaskSketcherElements.cpp

```diff
@@ -667,90 +667,90 @@ void ElementView::contextMenuEvent(QContextMenuEvent* event)
                  true)
     CONTEXT_ITEM("Constraint_PointOnObject",
-                 "Point on Object",
+                 "Point on object",
                  "Sketcher_ConstrainPointOnObject",
                  doPointOnObjectConstraint,
                  true)
     CONTEXT_ITEM("Constraint_Horizontal",
-                 "Horizontal Constraint",
+                 "Horizontal constraint",
                  "Sketcher_ConstrainHorizontal",
                  doHorizontalConstraint,
                  true)
     CONTEXT_ITEM("Constraint_Vertical",
-                 "Vertical Constraint",
+                 "Vertical constraint",
                  "Sketcher_ConstrainVertical",
                  doVerticalConstraint,
                  true)
     CONTEXT_ITEM("Constraint_Parallel",
-                 "Parallel Constraint",
+                 "Parallel constraint",
                  "Sketcher_ConstrainParallel",
                  doParallelConstraint,
                  true)
     CONTEXT_ITEM("Constraint_Perpendicular",
-                 "Perpendicular Constraint",
+                 "Perpendicular constraint",
                  "Sketcher_ConstrainPerpendicular",
                  doPerpendicularConstraint,
                  true)
     CONTEXT_ITEM("Constraint_Tangent",
-                 "Tangent Constraint",
+                 "Tangent constraint",
                  "Sketcher_ConstrainTangent",
                  doTangentConstraint,
                  true)
     CONTEXT_ITEM("Constraint_EqualLength",
-                 "Equal Length",
+                 "Equal constraint",
                  "Sketcher_ConstrainEqual",
                  doEqualConstraint,
                  true)
     CONTEXT_ITEM("Constraint_Symmetric",
-                 "Symmetric",
+                 "Symmetric constraint",
                  "Sketcher_ConstrainSymmetric",
                  doSymmetricConstraint,
                  true)
     CONTEXT_ITEM(
-        "Constraint_Block", "Block Constraint", "Sketcher_ConstrainBlock", doBlockConstraint, true)
+        "Constraint_Block", "Block constraint", "Sketcher_ConstrainBlock", doBlockConstraint, true)
 
     CONTEXT_ITEM("Constraint_HorizontalDistance",
-                 "Horizontal Distance",
+                 "Horizontal dimension",
                  "Sketcher_ConstrainDistanceX",
                  doHorizontalDistance,
                  true)
     CONTEXT_ITEM("Constraint_VerticalDistance",
-                 "Vertical Distance",
+                 "Vertical dimension",
                  "Sketcher_ConstrainDistanceY",
                  doVerticalDistance,
                  true)
     CONTEXT_ITEM("Constraint_Length",
-                 "Length Constraint",
+                 "Length dimension",
                  "Sketcher_ConstrainDistance",
                  doLengthConstraint,
                  true)
     CONTEXT_ITEM("Constraint_Radiam",
-                 "Radiam Constraint",
+                 "Radius/diameter",

```

Comment:
```suggestion
                 "Radius or diameter",
```
Only if you accept the other twin suggestion as well.